packages/fxa-settings/src/components/UnitRow/index.stories.tsx

```diff
@@ -59,4 +60,32 @@ storiesOf('Components|UnitRow', module)
         </UnitRow>
       );
     }
-  );
+  )
+  .add('with modal-triggering secondary CTA', () => {
+    const [modalRevealed, revealSecondaryModal, hideModal] = useBooleanState();
+    return (
+      <UnitRow
+        header="Display name"
+        headerValue={null}
+        {...{
+          secondaryCtaText: 'Disable',
```

Comment:
It looks like `secondaryCtaText` is set to "Disable" by default, so I think you can take this out.